script/eslint-plugin-va/rules/axe-e2e-tests.js

```diff
@@ -0,0 +1,37 @@
+const MESSAGE =
+  'e2e Tests must include at least one axeCheck call. Source code can be found here: ./src/platform/testing/e2e/nightwatch-commands/axeCheck.js';
+
+const rule = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: MESSAGE,
+      category: 'best practices',
+      recommended: false,
+    },
+  },
+  create: context => {
+    let axeCheckCount = 0;
+
+    const checkAxeCountForReport = node => {
+      if (axeCheckCount === 0) {
+        const message = `${MESSAGE}`;
+        context.report({
+          node,
+          message,
+        });
+      }
+    };
+
+    return {
+      MemberExpression: node => {
+        if (node.property.name === 'axeCheck') {
+          axeCheckCount++;
+        }
+      },
+      'Program:exit': checkAxeCountForReport,
```

Comment:
Where is the message being displayed? at the bottom? can you attach a screenshot of the results/tests?

Reply:
![image](https://user-images.githubusercontent.com/6521723/81980433-7027e900-95e3-11ea-8e95-8a4e603d28a4.png)